Add MongoDB Parrot

OliverVermeulen opened the pull request.

jmhobbs commented:
Did that do what you intended?  I don't see a stickbugparrot.gif in the files, only a mongodb logo parrot.

OliverVermeulen commented:
Hello John, I removed the stickbugparrot.gif as it was to big and didn't
really fit with the tone of Party Parrot so I added the mongodbparrot.gif

Thank you
Oliver

On Tue, 18 Oct 2022, 16:37 John Hobbs, ***@***.***> wrote:

> Did that do what you intended? I don't see a stickbugparrot.gif in the
> files, only a mongodb logo parrot.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jmhobbs/cultofthepartyparrot.com/pull/611#issuecomment-1282500896>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARG2BWJ2RBLDP4FDWT6TXR3WD2Y2BANCNFSM6AAAAAARH4QAKQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

jmhobbs commented on parrots.yaml:
```diff
@@ -537,3 +537,5 @@
 - gif: dizzyparrot.gif
   name: Pixelated parrot dizzy after spinning
+- gif: mongodbparrot.gif
+  name: MongoDB Parrot
```
Can you pop a newline at the end of the file here?  Thanks!

jmhobbs commented:
Perhaps consider putting the logo behind the parrot in the style of these other logo parrots?
![https://cultofthepartyparrot.com/parrots/hd/reactparrot.gif](https://cultofthepartyparrot.com/parrots/hd/reactparrot.gif)![https://cultofthepartyparrot.com/parrots/hd/innersourceparrot.gif](https://cultofthepartyparrot.com/parrots/hd/innersourceparrot.gif)![https://cultofthepartyparrot.com/parrots/hd/phparrot.gif](https://cultofthepartyparrot.com/parrots/hd/phparrot.gif)

jmhobbs commented:
Amazing! Thank you for your contribution!
